src/agenda/index.js

```diff
@@ -135,7 +137,9 @@ export default class AgendaView extends Component {
     // When user touches knob, the actual component that receives touch events is a ScrollView.
     // It needs to be scrolled to the bottom, so that when user moves finger downwards,
     // scroll position actually changes (it would stay at 0, when scrolled to the top).
-    this.setScrollPadPosition(this.initialScrollPadPosition(), false);
+    if (!this.props.shouldOpenCalendar) {
```

Comment:
prevent an animation conflict on initial render so that it either sets initial position or open the calendar